src/main.rs

```diff
@@ -102,13 +102,14 @@ use std::sync::atomic::{ AtomicBool, Ordering, ATOMIC_BOOL_INIT };
 use tls::TlsOption;
 
 docopt!(Args derive Debug, "
-Usage: foxbox [-v] [-h] [-l <hostname>] [-p <port>] [-w <wsport>] [-r <url>] [-i <iface>] [-t <tunnel>] [-s <secret>] [--disable-tls] [--remote-name <hostname>] [-c <namespace;key;value>]...
+Usage: foxbox [-v] [-h] [-l <hostname>] [-p <port>] [-w <wsport>] [-d <profile_dir>] [-r <url>] [-i <iface>] [-t <tunnel>] [-s <secret>] [--disable-tls] [--remote-name <hostname>] [-c <namespace;key;value>]...
 
 Options:
     -v, --verbose            Toggle verbose output.
     -l, --local-name <hostname>    Set local hostname. Linux only. Requires to be a member of the netdev group.
     -p, --port <port>        Set port to listen on for http connections. [default: 3000]
     -w, --wsport <wsport>    Set port to listen on for websocket. [default: 4000]
+    -d, --profile <path>     Set profile path [default: $HOME/.local/share/foxbox]
```

Comment:
Can we use `$XDG_DATA_HOME` instead of another command line option?

Check https://github.com/fxbox/foxbox/blob/d8ccbfa0da2d7c1a0fb74c38fd481210f3216e98/src/profile_service.rs#L28

@fabricedesre wdyt?